tslib/src/functions/check-marker/check-marker.v1.ts

```diff
@@ -107,39 +113,67 @@ export class CheckMarkerV1 {
     }
 
     const toReturn: MarkingData = {
-      answers: parsedAnswersJson,
+      answers: sortedAnswers,
       formQuestions: checkForm,
       results: []
     }
     return toReturn
   }
 
-  private markCheck (markingData: MarkingData): Mark {
-    const results: Mark = {
+  private markCheck (markingData: MarkingData, checkCode: string): CheckResult {
```

Comment:
is there a chance at that stage that markingData don't have the appropriate object structure?

Reply:
@GeorgeChatzigiannis - is there a specific path that would cause that?